interop/Cargo.toml

```diff
@@ -8,6 +8,8 @@ publish = false
 default = ["protobuf-codec"]
 protobuf-codec = ["protobuf", "grpcio/protobuf-codec", "grpcio-proto/protobuf-codec"]
 protobufv3-codec = ["protobufv3", "grpcio/protobufv3-codec", "grpcio-proto/protobufv3-codec"]
+_secure = []
+openssl = ["_secure", "grpcio-sys/openssl"]
```

Comment:
Why need these new features?

Reply:
Changed that to:
openssl = []
Need this because of this error:
   Compiling interop v0.1.0 (/grpc-rs/interop)
error: unexpected `cfg` condition value: `openssl`
  --> interop/tests/tests.rs:56:11
   |
56 | #[cfg(not(feature = "openssl"))]
   |           ^^^^^^^^^^^^^^^^^^^
   |
   = note: expected values for `feature` are: `default`, `protobuf`, `protobuf-codec`, `protobufv3`, and `protobufv3-codec`
   = help: consider adding `openssl` as a feature in `Cargo.toml`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
   = note: `-D unexpected-cfgs` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unexpected_cfgs)]`

error: unexpected `cfg` condition value: `openssl`
  --> interop/tests/tests.rs:60:11
   |
60 | #[cfg(not(feature = "openssl"))]
   |           ^^^^^^^^^^^^^^^^^^^
   |
   = note: expected values for `feature` are: `default`, `protobuf`, `protobuf-codec`, `protobufv3`, and `protobufv3-codec`
   = help: consider adding `openssl` as a feature in `Cargo.toml`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration

error: could not compile `interop` (test "tests") due to 2 previous errors

Reply:
Instead of adding new features, you can just delete those feature conditions as they are for some very old versions, it may be fixed now (giving it's always evaluated as true at the moment).

Reply:
Fixed!